components/Starknet/modules/architecture_and_concepts/pages/Smart_Contracts/cairo-builtins.adoc

```diff
@@ -0,0 +1,26 @@
+[id="cairo-builtins"]
+= Cairo builtins
+
+_Builtins_ in Cairo are predefined optimized low-level execution units that the Cairo VM refers to in order to perform predefined computations that are expensive to perform in standard Cairo. Builtins enhance the functionality of the Cairo VM, enabling you to perform certain tasks, such as using the Poseidon hash, range-checks, or ECDSA signature verifications, more efficiently, using less gas.
```

Comment:
> using less gas

using less trace cells (the unit in which we usually measure proving-complexity).

Reply:
PTAL